src/plugins/Plugin.js

```diff
@@ -89,6 +89,18 @@ module.exports = class Plugin {
     }
   }
 
+  setMetaFromTargetForm () {
+    const el = findDOMElement(this.opts.target)
+    const formMetaData = getMetaFromForm(el)
+    if (formMetaData) {
+      this.core.log('Adding metadata from form:')
+      this.core.log(formMetaData)
+      this.core.setMeta(formMetaData)
+    } else {
+      this.core.log('Couldn’t extract metadata from form')
+    }
+  }
```

Comment:
I'm not sure about putting `setMetaFromTargetForm` on the Plugin class, since it's not for use by every plugin…Maybe the plugins could manually do

```js
this.core.setMeta(getMetaFromForm(this.target))
```

in their `install()` methods? that's not a _lot_ of code to duplicate.